internal/npm_install/generate_build_file.ts

```diff
@@ -1101,26 +1101,6 @@ alias(
     actual = "contents",
 )
 `;
-
-  let mainEntryPoint = resolvePkgMainFile(pkg);
-
-  // add an `npm_umd_bundle` target to generate an UMD bundle if one does
-  // not exists
-  if (mainEntryPoint && !findFile(pkg, `${pkg._name}.umd.js`)) {
-    result +=
-        `load("@build_bazel_rules_nodejs//internal/npm_install:npm_umd_bundle.bzl", "npm_umd_bundle")
-
-npm_umd_bundle(
```

Comment:
npm_umd_bundle does not work with directory artifacts so no point in stamping it out

Reply:
would a user manually write one in their own BUILD file then?

Reply:
ts_library & ng_module are not compatible with exports_directories_only so the whole UMD concat js path is not as load bearing with the flag set.

If the user needed to make a UMD bundle out of an npm package with exports_directories_only set they could just use the `browserify-wrapped` npm tool directly with npm_package_bin.

The `npm_umd_bundle` rule has an explicit failure if any of the sources are directories.

```
    for s in sources:
        if s.is_directory:
            fail("npm_umd_bundle does not work with directory artifacts")
```

in `internal/npm_install/npm_umd_bundle.bzl`